DuckDuckGo/HomeViewController.swift

```diff
@@ -51,7 +53,14 @@ class HomeViewController: UIViewController {
         let feature = HomeRowOnboarding()
         if !feature.showNow() {
             hideCallToAction()
-        }    
+        }
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+         DispatchQueue.main.asyncAfter(deadline: .now() + 1) { [weak self] in
+            self?.viewLoadingHasSettled = true
+        }
```

Comment:
@subsymbolic why is it necessary to do this async?

Reply:
Can't do it immediately because we need the UI to settle. What would you prefer for dispatching after a 1second delay?

Reply:
I'm just not sure it's necessary. If I comment out the async part and just set the flag I still don't see the glitch.

Reply:
Ahh good point I don't need it anymore. Deleted!